frontend/nc-games/src/App.js

```diff
@@ -13,6 +13,7 @@ function App() {
             <Route path="/" element={<Welcome />} />
             <Route path="/reviewlist" element={<Reviewlist />} />
             <Route path="/categories" element={<Categories />} />
+            <Route path="/reviewlist/:category" element={<Reviewlist />}/>
```

Comment:
Your reviewlist component is still named inconsistently (the reviewlist file itself is not capitalised, but the places in which it is being called are). Same thing with navigation - and it means your code won't run when cloned. Please make sure this is consistent across the app!